lib/ansible/config/base.yml

```diff
@@ -886,6 +886,21 @@ DEFAULT_NETCONF_PLUGIN_PATH:
   ini:
   - {key: netconf_plugins, section: defaults}
   type: pathspec
+DEFAULT_WIN_ASYNC_STARTUP_TIMEOUT:
```

Comment:
Actually we've tried to stop using the `DEFAULT_` prefix on new items (with varying degrees of success) ;)